styles/wallet.module.css

```diff
@@ -1,10 +1,13 @@
 .walletGrid {
   display: grid;
-  grid-template-columns: repeat(auto-fit, minmax(160px, 1fr));
+  grid-template-columns: repeat(auto-fill, 160px);
```

Comment:
Using the same width as a card will be (see fixed width below) prevents a grid layout shift if there aren't enough cards for the first row.